apps/xprof_gui/priv/src/components/monitoring/Monitoring/Monitoring.jsx

```diff
@@ -50,13 +66,15 @@ class Monitoring extends React.Component {
       expandGraphPanel,
       shrinkGraphPanel,
       calleeClick,
+      running,
     } = this.props;
+    const { snapshot } = this.state;
     return (
       <div>
         <GraphPanel
           key={mfa[3]}
           mfa={mfa}
-          dps={data}
+          dps={running ? data : snapshot}
```

Comment:
with the update that polling is skipped if `!running` do we still need a snapshot? (as data won't be updated and no re-rendering will be trigger, if I understand correctly)

what I noticed is that if I remove this snapshot solution, when refreshing the page while tracing is stopped there will be an empty graph until tracing is resumed and data is fetched again. But as far as I know data for a given MFA is already stored between removing a monitor and adding it back, so that mechanism could be reused.